### DISCUSSION TITLE
Where is the M1 Mac chipset supported Apache Superset ?

### DISCUSSION THREAD

**Discussion by crosstowntraffic** [apache/superset#20271] [Q&A / Help]
After running docker pull apache/superset and having it download to my Mac I then ran into this warning --
 Downloaded newer image for apache/superset:latest
docker.io/apache/superset:latest
**docker run -d -p 8080:8088 --name superset apache/superset 
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested**
Yes, I'm running an M1 Mac
<img width="382" alt="Screenshot 2022-06-05 at 15 52 09" src="https://user-images.githubusercontent.com/85983375/172056527-cdf593cf-1c74-4fc8-ad16-19c17fd3edb7.png">
M1 Macs have been selling in the millions since 2020
Has Apache Superset catered to this market yet?
None of the pulls make note of updating the master for ARM64 chipsets.
Am I out of luck with Apache Superset this summer again?

**Comment by e-compagno** [comment]
I get a similar error
```
superset-worker The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```
with an Apple M2, using docker compose

**Comment by vitormattioli** [comment]
Is there any updates about this issue?

**Comment by mathenymi** [comment]
Can we get an update to this issue?

**Comment by mjtrac** [comment]
On 1/30/24, on a Mac M1, I executed docker pull apache/superset and got "no matching manifest for linux/arm64/v8 in the manifest list entries."

I'd previously downloaded it from github onto the same machine, but following the instructions at https://superset.apache.org/docs/installation/installing-superset-using-docker-compose gave me warnings about CYPRESS_CONFIG and SCARF_ANALYTICS not being set and errors on superset-tests-worker and superset-init

**Comment by sfirke** [comment]
There is now some guidance published in the docs about [how to run Superset in docker containers on an M1 or M2 Mac](https://superset.apache.org/docs/installation/docker#on-arm-builds-and-working-with-apple-silicon).  If people have additions or feedback on that, please share it so we can improve the guidance for users with such equipment.

**Comment by rusackas** [comment]
It all works now! :D
